compiler/z/codegen/OMRLinkage.cpp

```diff
@@ -2840,7 +2840,9 @@ OMR::Z::Linkage::getLastMaskedBit(int16_t mask)
 bool
 OMR::Z::Linkage::isOSLinkageType()
    {
-   return (self()->getExplicitLinkageType() & TR_SystemOS_MASK) != 0;
+   // #define TR_SystemOS_MASK  0x40
+   // zOS Type 1 linkages (non-Java) - all have bit 0x40 set
+   return (self()->getExplicitLinkageType() & 0x40) != 0;
    }
```

Comment:
This doesn't appear to be used anywhere. Can we fold the API and get rid of this magic constant logical AND?

Reply:
Do you mean that `isOSLinkageType` isn't used anywhere? There's a use for it here: https://github.com/eclipse/omr/blob/d19d2fff4fab1bc47f7288c029793468969e9ef3/compiler/z/codegen/SystemLinkage.cpp#L299-L303

Reply:
You're totally right. I missed this in my search. However looking at the return values of `getExplicitLinkageType` the only enum value that would fit this criteria is `TR_SystemFastLink`, and I don't see any uses of that enum value since we don't support fast link in OMR/OpenJ9. So the `isOSLinkageType` should always return false from what I'm reading. Given that I think we should still fold it away as `true` in this one use case.

The stack slots to where we should save/restore FPRs should be 16 byte aligned for all our Z system linkages in OMR.

Reply:
@dchopra001 I think we still need to address this one.

Reply:
9a287b7